packages/crayons-core/src/components/button/button.scss

```diff
@@ -1,50 +1,58 @@
 /**
   @prop --fw-button-min-width: minimum width for the button
   @prop --fw-button-label-vertical-padding: vertical padding for the button label
+  @prop --fw-button-height: The height of the normal button.
+  @prop --fw-button-small-height: The height of the small button.
+  @prop --fw-button-border-radius: The height of the small button.
```

Comment:
if this is for only small button, can we also add `small` to the css variable ?

Reply:
Its a typo, modified it.